lib/imports/emails.js

```diff
@@ -40,7 +40,7 @@ export const registrationInfoHTML = `
       Deadline: <strong>11:59 PM ${registrationCloseDate}</strong>
     </li>
     <li>
-      Purchase ticket code(s) at <a target="_blank" href="https://commerce.cashnet.com/TheGreatPuzzleHunt2020">https://commerce.cashnet.com/TheGreatPuzzleHunt2020</a>.<br />
+      Purchase ticket code(s) at <a target="_blank" href="https://commerce.cashnet.com/TheGreatPuzzleHunt${eventYear}">https://commerce.cashnet.com/TheGreatPuzzleHunt${eventYear}</a>.<br />
```

Comment:
Hrm, this looks suspect. It's a normal string with quotes (`"`) instead of backticks, so I don't think the `${eventYear}` will be replaced with the value. Can you test this?

Reply:
Wait, never mind. I read through the whole file and I see the context better now. You probably tested it, but if not it'd be great to quickly. Sorry for the confusion!